isthmus/src/main/java/io/substrait/isthmus/expression/RexExpressionConverter.java

```diff
@@ -80,11 +93,49 @@ public Expression visitRangeRef(RexRangeRef rangeRef) {
 
   @Override
   public Expression visitFieldAccess(RexFieldAccess fieldAccess) {
-    throw new UnsupportedOperationException("RexFieldAccess not supported");
+    if (fieldAccess.getReferenceExpr() instanceof RexCorrelVariable) {
+      int stepsOut = relVisitor.getFieldAccessDepth(fieldAccess);
+
+      return FieldReference.newRootStructOuterReference(
+          fieldAccess.getField().getIndex(),
+          TypeConverter.convert(fieldAccess.getType()),
+          stepsOut);
+    }
+    throw new UnsupportedOperationException(
+        "RexFieldAccess for other than RexCorrelVariable not supported");
   }
 
   @Override
   public Expression visitSubQuery(RexSubQuery subQuery) {
+    Rel rel = relVisitor.apply(subQuery.rel);
+
+    if (subQuery.getOperator() == SqlStdOperatorTable.EXISTS) {
```

Comment:
This looks like it might be better as a switch statement. (?)

Reply:
switch statement requires compile time constant. Here `SqlStdOperatorTable.EXISTS` is run-time constant. Seems I could not change to a switch statement.